pyiron_contrib/atomistic/atomicrex/atomicrex_job.py

```diff
@@ -0,0 +1,127 @@
+"""
+Pyiron interface to atomicrex
+"""
+import xml.etree.ElementTree as ET
+from xml.dom import minidom
+import posixpath
+import os
+import subprocess
+
+from pyiron_base import GenericJob, Settings, PyironFactory
+
+from pyiron_contrib.atomistic.atomicrex.general_input import GeneralARInput
+from pyiron_contrib.atomistic.atomicrex.structure_list import ARStructureList
+from pyiron_contrib.atomistic.atomicrex.potential_factory import ARPotFactory
+from pyiron_contrib.atomistic.atomicrex.output import Output
+
+
+s = Settings()
+
+## Class defined for future addition of other codes
+## Not sure which functionality (if any) can be extracted yet, but a similar pattern is followed in other pyiron modules
+class PotentialFittingBase(GenericJob):
```

Comment:
Like it, probably once #51 is in, we want to include it here.  Can you already move that class to something like pyiron_contrib/atomistic/job/potential_fitting.py?